packages/sandbox/src/app/sandbox/docs/list-item.doc.tsx

```diff
@@ -8,6 +8,12 @@ const ICON_NAME = 'Area';
 const LONG_DESCRIPTION =
   'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.';
 const ON_PRESS = () => Alert.alert('item pressed');
+const ASYNC_ON_PRESS = async () =>
+  new Promise((resolve) =>
+    setTimeout(() => {
+      resolve(undefined);
+    }, 2000)
+  );
```

Comment:
We should create a folder with utilities...

I have my own sleep function too 🙈 in `streamline/packages/sandbox/src/app/sandbox/docs/card-summary.doc.tsx`

```
  const sleep = async () =>
    new Promise((resolve) =>
      setTimeout(() => {
        resolve(undefined);
      }, 2000)
    );
 ```

Reply:
same for on_press with an alert?